lua/anywise_reg/keybinds.lua

```diff
@@ -48,36 +48,26 @@ local function setup_yank_keymaps(prefix)
     end
 end
 
-local function setup_paste_keymaps(prefix)
-    local lhs = prefix..config.paste_key
-    local rhs = '<Cmd>lua require("anywise_reg.handlers").handle_paste(\''..prefix..'\')<CR>'
-    set_keymap(lhs, rhs)
-end
-
-local function setup_paste_behind_keymaps(prefix)
-    local lhs = prefix .. config.paste_behind_key
-    local rhs =
-        '<Cmd>lua require("anywise_reg.handlers").handle_paste_behind(\'' ..
-            prefix .. '\')<CR>'
+local function setup_paste_keymaps(prefix, key, operator)
+    local lhs = prefix .. key
+    local rhs = '<Cmd>lua require("anywise_reg.handlers").handle_paste(\'' ..
+                    prefix .. '\',\'' .. operator .. '\')<CR>'
```

Comment:
You could use `format_str_args` here, see https://github.com/AckslD/nvim-anywise-reg.lua/blob/main/lua/anywise_reg/keybinds.lua#L45